src/utils/geojson-utils.js

```diff
@@ -41,6 +41,10 @@ const isValidBBox = boundingBox => {
   if (!boundingBox || boundingBox.length != 4) { return false }
 
   const [minLong, minLat, maxLong, maxLat] = boundingBox
+
+  if (boundingBox.some(x => x === null)) { 
```

Comment:
Thank you for fixing this.  Did you look into why our service is returning nulls for the bounding box?

Reply:
Nope. It looks like there's some logic in the frontend for calculating these as well.

This seemed like the most appropriate fix for the immediate problem.